backend/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:slim
+
+WORKDIR /app
+
+COPY package.json .
+
+COPY package-lock.json .
+
+RUN npm install && npm install -g nodemon
+
+ARG NODE_ENV
+ENV NODE_ENV=${NODE_ENV}
+
+COPY . .
+
+CMD ["npm", "run", "server"]
```

Comment:
```suggestion
CMD ["npm", "start"]
```

Reply:
Another comment, can we remove yarn.lock and start using npm everywhere with package-lock.json? I don't see any reason to use yarn anymore